.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Release checklist
+about: Checklist for a new release
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+- [ ] Document all important changes in Changelog.md
+- [ ] Set the release date in Changelog.md
+- [ ] Set the current release version and data in Changelog.md
```

Comment:
We don't have the current release version and data in Changelog.md

Reply:
Typo. Should be "README.md"